models/src/main/java/io/github/kgress/scaffold/models/enums/ScreenResolution.java

```diff
@@ -0,0 +1,56 @@
+package io.github.kgress.scaffold.models.enums;
+
+public enum ScreenResolution {
+
+    SIZE_800x600(800, 600),
```

Comment:
might be worth adding javadoc to the class to say that these are just some predefined choices and remind the user that they can create their own custom resolution if wanted.

Reply:
Good suggestion. Adding now.